### DISCUSSION TITLE
Is there a way to turn of the unused-value error?

### DISCUSSION THREAD

**Discussion by tjt22** [bitcraze/discussions#459] [Q&A] [answered]
Good afternoon,

Sometimes if I run my code and a variable doesn't have a value, I get the following error: `-Werror=unused-value`. To fix this, I give the variables in question random values and then the error clears, but this is obviously very tedious. Is there a way to get rid of this warning?

**Comment by krichardsson** [comment, marked as answer]
The easiest way is to either build in debug mode, or remove [the setting in the make file](https://github.com/bitcraze/crazyflie-firmware/blob/4f481aba69f577f29030d03668cfabd9c2b852b2/Makefile#L116)

```make
# Original setting that fails on warnings
# ARCH_CFLAGS += -Os -Werror

# Setting that does not fail on warnings
ARCH_CFLAGS += -Os
```
